include/irr/asset/IVirtualTexture.h

```diff
@@ -751,6 +762,22 @@ class IVirtualTexture : public core::IReferenceCounted, public IVirtualTextureBa
 
     virtual SViewAliasTextureData createAlias(const SMasterTextureData& _addr, E_FORMAT _viewingFormat, const IImage::SSubresourceRange& _subresRelativeToMaster) = 0;
 
+    //! @returns pointer to reserved space for allocators
+    uint8_t* copyVirtualSpaceAllocatorsState(uint32_t _count, pg_tab_addr_alctr_t* _dstArray)
+    {
+        _count = std::min(_count, m_pageTable->getCreationParameters().arrayLayers);
+        const uint32_t bufSz = m_pageTableLayerAllocators[0].get_total_size();
+        const uint32_t resSpcPerAlctr = pg_tab_addr_alctr_t::reserved_size(m_pageTableLayerAllocators[0].get_total_size(), m_pageTableLayerAllocators[0]);
+        uint8_t* reservedSpc = reinterpret_cast<uint8_t*>( impl::alloc_and_copy(resSpcPerAlctr*_count, m_pgTabAddrAlctr_reservedSpc) );
```

Comment:
ok this was kindly badly phrased
https://github.com/buildaworldnet/IrrlichtBAW/blob/shader_pipeline/include/irr/core/alloc/PoolAddressAllocator.h#L42

what it actually means is that the underlying data buffer (memory, file, GPU buffer, image, whatever) whose ranges the address allocator allocates should be handled by the user.

The move constructors actually copied the state in the reserved space properly, so there's no need for a `alloc_and_copy` just an alloc